docs/scripts/build_documentation.py

```diff
@@ -0,0 +1,155 @@
+"""Utility for building the documentation."""
+import argparse
+import os
+import pathlib
+from subprocess import check_call, run
+
+from build_examples import build_examples
+from check_links import check_links
+from utils import adjust_pictures
+
+from baybe.telemetry import VARNAME_TELEMETRY_ENABLED
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-e",
+    "--calculate_examples",
```

Comment:
I find the word "calculate" a bit weird in this context. Why not simply "run" or "build"?

Reply:
changed